.prettierignore

```diff
@@ -14,6 +15,7 @@ coverage
 # be done in the future as we don't want to update the Markdown file
 # contents right now
 pages/**/*.md
+pages/_app.mdx
```

Comment:
why?

Reply:
Running `prettier:fix` will update `_app.mdx`:
![Screenshot 2023-05-15 at 7 37 58 PM](https://github.com/nodejs/nodejs.org/assets/128208841/d59baf74-b5c1-4966-9023-5e5340d4baaa)

Reply:
Something might be wrong with your prettier configuration. Check prettier and MDX integration. It should be working, but maybe something broke.